templates/deployment.yaml

```
@@ -0,0 +1,155 @@
+{{- range $name, $values := .Values.Servers }}
+{{- $fullName := include "<CHARTNAME>.fullname" $ }}  # Use $ to access the root context
+{{- $releasePrefix := include "<CHARTNAME>.releasePrefix" $ }}
+{{- $_ := set $.Release "Prefix" $releasePrefix }}
```

You have to add `---` if you want to generate multiple resources from a single file
```suggestion
{{- $_ := set $.Release "Prefix" $releasePrefix }}
---
```

---

Good catch, Yes i forget to add